tests/common.go

```diff
@@ -1726,14 +1726,15 @@ func GetStorageNodes() ([]node.Node, error) {
 
 	storageNodes := []node.Node{}
 	nodes := node.GetStorageDriverNodes()
-
-	for _, node := range nodes {
-		devices, err := Inst().V.GetStorageDevices(node)
-		if err != nil {
-			return nil, err
-		}
-		if len(devices) > 0 {
-			storageNodes = append(storageNodes, node)
+	for _, eachNode := range nodes {
+		if node.IsStorageNode(eachNode) {
```

Comment:
this seems like a redundant method, can you delete this and replace it with node.GetStorageNodes

Reply:
This is not redundant , I am modifying already available function , which is GetStorageNodes() , 
I am basically checking if the node is storageNode , and then proceeding